VK_App/VK_App/Service/RealmService.swift

```diff
@@ -6,28 +6,28 @@ import RealmSwift
 /// Рeaлм сервис
 final class RealmService {
     // MARK: - Private Constants
-    
+
     private enum Constants {
         static let errorText = "Error"
     }
-    
+
     static let deleteIfMigration = Realm.Configuration(deleteRealmIfMigrationNeeded: true)
-    
+
     // MARK: - Public Methods
-    
+
     func saveToRealm<T: Object>(object: [T]) {
```

Comment:
```suggestion
    func saveToRealm<T: Object>(object: [T]) {
```
в важном есть файлик realmService, там есть статический метод по сохранению в реалм, по хорошему переделать на него, чтобы эта функция у тебя не была публичной